examples/adminapi.py

```diff
@@ -846,12 +846,14 @@ def example_list_offsets(a, args):
 
 
 def example_delete_records(a, args):
-    topic_partition_offset = []
     if len(args) == 0:
         raise ValueError(
-            "Invalid number of arguments for delete_records, expected at least 3, got 0")
+            "Invalid number of arguments for delete_records, expected at least 3 " +
```

Comment:
Add tests for these new validations and remove old tests which are incorrect now.